### DISCUSSION TITLE
Postgis version required for supported pgrouting versions

### DISCUSSION THREAD

**Discussion by benjaminjb** [pgRouting/pgrouting#2507] [General]
I want to check/verify something: according to the documentation `pgrouting` v3.x requires `postgis` 2.2 or greater. 

Is that the only relation between `pgrouting` and `postgis` versions?

I was expecting something like a compatibility matrix, but if the general rule is captured by that sentence ("`pgrouting` v3.x requires `postgis` 2.2 or greater"), then I guess we don't need a matrix.

**Comment by cvvergara** [comment]
Yeah, but now postgis 2.x is no longer supported.
Basically any postgis version would work as we use very basic functions of it
